### PR TITLE
Add entries for interpreter directive, hashbang, and shebang

### PR DESCRIPTION
**Pull request by mportman12**
Based on [Issue 381](https://github.com/redhat-documentation/supplementary-style-guide/issues/381)

@redhat-documentation/ccs-style-council please review

### CODE REVIEW

**Review by ndeevy — APPROVED** [review]
Looks good, @mportman12

**Review by bredamc — COMMENTED** [review]
Just a few minor comments :)

**Comment by mportman12** [comment]
Made @bredamc's suggested edits. Breda, @ndeevy, please review again.

**Review by bredamc — APPROVED** [review]
LGTM -- thanks, Miriam :)

**Review by bergerhoffer — COMMENTED** [review]
Just a few things!

**Comment by bredamc** [comment]
@mportman12 I agree with all of Andrea's comments :)

File: supplementary_style_guide/glossary_terms_conventions/general_conventions/h.adoc [hunk 1]
```diff
@@ -18,6 +18,15 @@
 
 *See also*:
 
+[[hashbang]]
+==== image:images/no.png[no] hashbang (noun)
+*Description*: Do not use to refer to a computer language construct that controls which interpreter parses and interprets instructions in a computer program. Instead, on first mention include both "interpreter directive" and "shebang", for example, "Interpreter directives, also known as shebangs".
+*Use it*: no
```
[the remark this answers is not in the document]
Comment (mportman12) [reply supplementary_style_guide/glossary_terms_conventions/general_conventions/h.adoc:24]: Fixed

File: supplementary_style_guide/glossary_terms_conventions/general_conventions/h.adoc [hunk 2]
```diff
@@ -18,6 +18,15 @@
...
+
+*Incorrect forms*:
+
+*See also*: xref:hashbang[hashbang], xref:shebang[shebang]
```
[the remark this answers is not in the document]
Comment (mportman12) [reply supplementary_style_guide/glossary_terms_conventions/general_conventions/h.adoc:28]: Fixed

File: supplementary_style_guide/glossary_terms_conventions/general_conventions/s.adoc [hunk 3]
```diff
@@ -397,6 +397,15 @@ Use all lower case, unless used in a title or at the beginning of a sentence.
 
 *See also*: xref:he[he]
 
+[[shebang]]
+==== image:images/yes.png[yes] shebang (noun)
+*Description*: A _shebang_ such as `#!/bin/bash` is a common term for an _interpreter directive_, which is a computer language construct that controls which interpreter parses and interprets instructions in a computer program. On first mention, include both "interpreter directive" and "shebang", for example, "Interpreter directives, also known as shebangs". Do not use "hashbang".
+*Use it*: yes
```
[the remark this answers is not in the document]
Comment (mportman12) [reply supplementary_style_guide/glossary_terms_conventions/general_conventions/s.adoc:403]: Fixed

File: supplementary_style_guide/glossary_terms_conventions/general_conventions/i.adoc [hunk 4]
```diff
@@ -567,6 +567,15 @@ _This feature can run only on Intel 64 processors_
 *See also*: xref:raw-interpreted-program[raw-interpreted program], xref:byte-compiled-program[byte-compiled program]
 
 // RHEL: Added "In Red Hat Enterprise Linux, an inventory is"
+[[interpreter-directive]]
+==== image:images/yes.png[yes] interpreter directive (noun)
+*Description*: An _interpreter directive_ such as `#!/bin/bash` is a computer language construct that controls which interpreter parses and interprets instructions in a computer program. This term is also commonly known as _shebang_. On first mention, include both "interpreter directive" and "shebang", for example, "Interpreter directives, also known as shebangs". Do not use "hashbang".
+*Use it*: yes
```
[the remark this answers is not in the document]
Comment (mportman12) [reply supplementary_style_guide/glossary_terms_conventions/general_conventions/i.adoc:573]: fixed

File: supplementary_style_guide/glossary_terms_conventions/general_conventions/i.adoc [hunk 5]
```diff
@@ -567,6 +567,15 @@ _This feature can run only on Intel 64 processors_
 *See also*: xref:raw-interpreted-program[raw-interpreted program], xref:byte-compiled-program[byte-compiled program]
 
 // RHEL: Added "In Red Hat Enterprise Linux, an inventory is"
+[[interpreter-directive]]
+==== image:images/yes.png[yes] interpreter directive (noun)
+*Description*: An _interpreter directive_ such as `#!/bin/bash` is a computer language construct that controls which interpreter parses and interprets instructions in a computer program. This term is also commonly known as _shebang_. On first mention, include both "interpreter directive" and "shebang", for example, "Interpreter directives, also known as shebangs". Do not use "hashbang".
```
[the remark this answers is not in the document]
Comment (mportman12) [reply supplementary_style_guide/glossary_terms_conventions/general_conventions/i.adoc:572]: Fixed

**Comment by mportman12** [comment]
When I tried to update this PR for Andrea's edits, I ran into some major merge conflicts due to the age of this PR. To avoid resolving those merge conflicts, I have created a new PR with the glossary entries and Andrea's incorporated changes: [PR#432](https://github.com/redhat-documentation/supplementary-style-guide/pull/432)
Please review in the new PR.

**Pull request closed without merging**